bioimageio/spec/model/v0_3/schema.py

```diff
@@ -419,13 +419,13 @@ def raise_on_weights_format_mismatch(self, raw_node, **kwargs):
 class KerasHdf5WeightsEntry(_WeightsEntryBase):
     bioimageio_description = "Keras HDF5 weights format"
     weights_format = fields.String(validate=field_validators.Equal("keras_hdf5"), required=True, load_only=True)
-    tensorflow_version = fields.StrictVersion()  # todo: required=True
+    tensorflow_version = fields.StrictVersion()
 
 
 class OnnxWeightsEntry(_WeightsEntryBase):
     bioimageio_description = "ONNX weights format"
     weights_format = fields.String(validate=field_validators.Equal("onnx"), required=True, load_only=True)
-    opset_version = fields.Integer()  # todo: required=True
+    opset_version = fields.Integer()
 
 
 class PytorchStateDictWeightsEntry(_WeightsEntryBase):
```

Comment:
Don't we need (optional) `pytorch_version` for the `PytorchStateDict` and `PytorchScript` entries as well?

Reply:
yes, see my comments below..

Reply:
Ok, I see. These are the files for 0.3...